Automattic/jetpack
# AI Extension: check the AI quota exceeded from the AI context

This pull request examines the `responseError` from the AI content to determine if the quota has been surpassed. This enables the upgrade banner to be displayed immediately whenever a user requests a suggestion.
Fixes #

## Proposed changes:

* AI Extension" check the AI quota exceeded from the AI context

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Set the environment to get a `429` error
* Confirm that after request a suggestion, the app shows the upgrade banner

https://github.com/Automattic/jetpack/assets/77539/35fbffa5-fd75-44f1-a4b4-67bf97ac683a
